ragaai_catalyst/tracers/agentic_tracing/utils/llm_utils.py

```
@@ -290,15 +305,16 @@ def sanitize_value(value):
     }
 
 
-def calculate_llm_cost(token_usage, model_name, model_costs):
+def calculate_llm_cost(token_usage, model_name, model_costs, model_custom_cost={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)